matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/session/search/SearchMessagesTest.kt

```diff
@@ -76,7 +76,7 @@ class SearchMessagesTest : InstrumentedTest {
 
     @Test
     fun sendTextMessageAndSearchPartOfItIncompleteWord() {
-        doTest(expectedNumberOfResult = 0) { cryptoTestData ->
+        doTest(expectedNumberOfResult = 2) { cryptoTestData ->
```

Comment:
Yes, something happened again and looks like logic changed to what it was before. So I think we can just close this PR, but in general I think it's not even necessary to test this behavior, since we basically test more Backend than client here